engine/battle/core.asm

```diff
@@ -2648,8 +2653,9 @@ SelectMenuItem:
 	jr z, .disabled
 	ld a, [wPlayerBattleStatus3]
 	bit 3, a ; transformed
-	jr nz, .dummy ; game freak derp
-.dummy
+	jr nz, .transformedMoveSelected
```

Comment:
```suggestion
	jr nz, .transformedMoveSelected ; pointless
```

Reply:
Please refer to the other review about this subroutine.

Reply:
This is independent of that. Any `jr [cc,] .label / .label` is pointless, and although they may not all need commenting, I think if there's any comment below this to explain what the actual code is for, then this `; pointless` comment is useful to make it clear that the jump has nothing to do with what the code does or was meant to do.